### PR TITLE
PSR-1

### PR DESCRIPTION
**Pull request by marcmascort**
Renamed constant __DEFAULT in uppercase to support PSR-1.

### CODE REVIEW

**Comment by fulopattila122** [comment]
I understand the importance of PSRs, however they are still optional.
This PR breaks existing functionality and would cause people to need to update their existing code bases.
So me as a maintainer have to ask myself what's the benefit vs. cost ratio here?
What are your thoughts on that? 🙇

**Comment by marcmascort** [comment]
First, good job with this library!
For me, the benefit is to use standards, and the cost is not to use them.
I am aware that for the people who are currently using the library, they will have to review the code to update.
In my case, for example, if the library does not use standards, I will not use it. That is why I have preferred to contribute with this change, before discarding the library or use my own fork.

**Comment by fulopattila122** [comment]
OK, it makes sense. As a solution I can release it as v3.0, which as of semver.org means breaking change. Most likely people have sane version constraints in their composer thus won't upgrade to v3 without considering the BC.

Since this library is not likely to change a lot in the future, it's still possible to maintain v2.x and v3.x branches parallel.

I'll do this in the next few days - please keep this PR open until then.

**Comment by marcmascort** [comment]
Thanks!
When you tag v3.0 I will review the PR of `enum-eloquent` https://github.com/artkonekt/enum-eloquent/pull/4

File: UPGRADE-2.0.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ In order to be more straightforward, the following naming changes have been done
 ## Removed Features
 
 - strict comparison (`===`) has been removed (is always `==`)
-- `toArray()` never returns the '__default' key.
```
Comment (fulopattila122) [inline UPGRADE-2.0.md]: I'm preparing version 3.0 as discussed. Please revert these changes to this file, as the 2.x versions won't be affected.

File: docs/README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ class Status extends \Konekt\Enum\Enum {
```
Comment (fulopattila122) [inline docs/README.md:25]: We will also need to maintain two parallel versions of the [Documentation](https://artkonekt.github.io/enum/#/), something like the [PHPUnit Documentation](https://phpunit.readthedocs.io/en/8.0/). (screenshot below)

Enum's doc is built with [docsify](https://docsify.js.org/) + github pages. Are you aware of any versioning solution for docsify?

image

**Review by fulopattila122 — CHANGES_REQUESTED** [review]
Please change the target to the `3.0` branch.

File: UPGRADE-2.0.md (hunk 1 above)
Comment (marcmascort) [reply UPGRADE-2.0.md]: Revert: https://github.com/artkonekt/enum/pull/3/commits/85b197ab3fa5ea04d0c43fcf587a6fc81c1ecf6e

File: docs/README.md (hunk 2 above)
Comment (marcmascort) [reply docs/README.md:25]: Docsify supports [Multiple covers](https://docsify.js.org/#/cover?id=multiple-covers). It is useful for languages or versions.

Comment (fulopattila122) [reply docs/README.md:25]: Cool. Problem 1/2 solved. Now just have to figure out how to integrate that with github-pages.

**Comment by marcmascort** [comment]
- Changed target to `3.0` branch
- Added the base for docsify with multiple covers

**Comment by fulopattila122** [comment]
As of now v3.0.0 has been released.

Documentation has been moved to:
[Doc 3.0](https://konekt.dev/enum/3.0)
[Doc 2.2](https://konekt.dev/enum/2.2)

Thanks for the suggestion ;)
Cheers

**Comment by marcmascort** [comment]
Thanks @fulopattila122 
I will review the PR https://github.com/artkonekt/enum-eloquent/pull/4 this week
